scribus/plugins/tools/hunspellcheck/hunspellpluginimpl.cpp

```diff
@@ -181,7 +181,7 @@ bool HunspellPluginImpl::parseTextFrame(StoryText *iText)
 			wf.replacements = hspellerMap[wordLang]->suggest(word);
 			wordsToCorrect.append(wf);
 		}
-		currPos = iText->nextWord(wordStart);
+		currPos = iText->endOfWord(wordStart);
```

Comment:
As we agreed on https://github.com/HOST-Oman/scribus/issues/166 I will not remove any unused code.